fern/docs-config/definition/docs.yml

```diff
@@ -65,11 +65,16 @@ types:
       accentPrimary: optional<string>
 
   NavbarLink:
+    union:
+      primary: NavbarPrimaryLink
+      secondary: NavbarSecondaryLink
```

Comment:
@kafkas to confirm you want the user here to specify: 

```
navbar-links: 
  - type: primary
    url: https://myurl.com
    text: Schedule a meeting!
  - type: secondary
     url: 
     text: 
```

Reply:
That's correct